TestingDangerApp/ViewController.swift

```diff
@@ -1,15 +1,16 @@
 import UIKit
 
 class ViewController: UIViewController {
-    lazy var label: UILabel = {
+    
+    lazy var label:UILabel = {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/45c1387bfb9d68560161334ef339af285201ad18/TestingDangerApp/ViewController.swift#L5"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>ViewController.swift:5</code></td>
    </tr>
  </tbody>
</table>